lxd/instance/qemu/vm_qemu.go

```diff
@@ -1007,8 +1050,9 @@ ConditionPathExists=/dev/virtio-ports/org.linuxcontainers.lxd
 Type=oneshot
 RemainAfterExit=yes
 ExecStartPre=-/sbin/modprobe 9pnet_virtio
-ExecStartPre=/bin/mkdir -p /run/lxd_config
-ExecStart=/bin/mount -t 9p config /run/lxd_config
+ExecStartPre=/bin/mkdir -p /run/lxd_config/9p
+ExecStartPre=/bin/chmod 0700 /run/lxd_config/
+ExecStart=/bin/mount -t 9p config /run/lxd_config/9p -o access=0
```

Comment:
Can you split the unit changes to a separate commit as that's not directly related.

Reply:
Done